pixart.js

```diff
@@ -1 +1,25 @@
 
+$("button").on("click", function(e) {
+  $(".brush").css("background", $("input").val());
+  e.preventDefault();
+})
+
+$(document).on("keypress", function(e) {
```

Comment:
I would look into the "submit" event and using that on the `<form>`. That way you don't need to worry about listening for a particular keypress.